### ISSUE TITLE
Implement Accessible Captcha

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4058]
Exisiting Captcha is not accessible.
Accessible Captcha is a requirement of the WCAG.

Discussion to move to hCaptcha, which has accessibility options. As part of that move, these need to be enabled and tested.

**Comment by StephDriver** [comment]
- [ ] when this is done, consider closing #3945 . That issue needs to remain open for now because it is being used to track the work done back to Site Improve A11y results.  Fixing this issue should fix that one.

**Comment by StephDriver** [comment]
Some have moved to hCapture, make sure all have moved across.

**Comment by joemull** [comment]
See https://github.com/BirkbeckCTP/bau/issues/39

**Comment by joemull** [comment]
We've now moved everyone to hcaptcha. Is there any particular way you wanted to test them @StephDriver ?

**Comment by StephDriver** [comment]
@joemull I think a random sample would probably be the way to go.

**Comment by joemull** [comment]
So I'd do one of each theme:
- [ ] OLH: https://journals.librarypublishing.arizona.edu/tbtr/login/
- [ ] Clean: https://zif.tujournals.ulb.tu-darmstadt.de/login/
- [ ] Material: https://traj.openlibhums.org/login/

**Comment by StephDriver** [comment]
I am having some a11y issues when testing, but I think those are not part of this change but part of the general need to update the accessibility of the register form - things like the Captcha heading being after the captcha.

**Comment by StephDriver** [comment]
Is leaving the recaptcha in place intentional?

```html
<textarea id="g-recaptcha-response-0tmwkkkfnlh" name="g-recaptcha-response" style="display: none;"></textarea>
<textarea id="h-captcha-response-0tmwkkkfnlh" name="h-captcha-response" style="display: none;"></textarea>
```

**Comment by StephDriver** [comment]
I set up an h-captcha a11y cookie, and the first test journal (OLH) recognises it fine. But I can't get either of the other two working.  I am going to wait a bit and then retest in case this is because it won't let me pass through h-captcha twice in a short time or something like that.

**Comment by StephDriver** [comment]
update. I can only get this to work on the first one journal I tested.  All subsequent hcaptcha requests have resulted in the inaccessible image game being displayed.

**Comment by StephDriver** [comment]
update 2. I have now tried on a new device. And again, it works for the first test (this time the Material test journal) but then does not allow subsequent use on another journal.

**Comment by StephDriver** [comment]
Summary:

I've tested this, as has Joe - we've shown between us that it works across the three test cases. The issues encountered (described above) appears to be to do with the way the hCaptcha a11y cookie works at their end (not allowing two requests in short succession as this might be non-genuine - well, yes, it wasn't a genuine attempt to register, it was a test!). 

So I am now closing this issue.
